apps/web/app/(use-page-wrapper)/(main-nav)/layout.tsx

```diff
@@ -1,6 +1,16 @@
+import { setUser as SentrySetUser } from "@sentry/nextjs";
+import { cookies, headers } from "next/headers";
+import React from "react";
+
+import { getServerSession } from "@calcom/features/auth/lib/getServerSession";
 import Shell from "@calcom/features/shell/Shell";
 
-const Layout = ({ children }: { children: React.ReactNode }) => {
+import { buildLegacyRequest } from "@lib/buildLegacyCtx";
+
+const Layout = async ({ children }: { children: React.ReactNode }) => {
+  const session = await getServerSession({ req: buildLegacyRequest(await headers(), await cookies()) });
+  if (session?.user?.id) SentrySetUser({ id: session.user.id });
```

Comment:
I figured this is the best place for authed routes in the dasheboard. cc @hbjORbj